api/authentication.go

```diff
@@ -22,28 +23,16 @@ func (a *Authenticator) Check(handle func(http.ResponseWriter, *http.Request)) h
 	return http.HandlerFunc(func(w http.ResponseWriter, r *http.Request) {
 		key := r.Header.Get(a.headerName)
 		if key == "" {
-			http.Error(w, "No authentication header provided", http.StatusForbidden)
-			log.Error(errors.New("client missing token"), log.Data{"header": a.headerName})
+			http.Error(w, genericError, http.StatusNotFound)
```

Comment:
The name `genericError` was a bit confusing as to its purpose. Consider calling it something like `notFoundErr`

Reply:
done